Changes to issue templates & codeowners (#1337)
celo-org/celo-blockchain: .github/ISSUE_TEMPLATE/issue.md

---
name: Issue
about: Feature/Need/Problem complete template
title: ''
labels: blockchain, needs-research
assignees: ''

---

## Description

> _Explain what's the need, what's the desired behaviour_

Reviewer: ```suggestion
_Explain what's the need, what's the desired behavior_
```

Spelling to make it consistent (see below)

> ## Current Behaviour

Reviewer: ```suggestion
## Current Behavior
```

Behaviour is the British spelling.  Nothing wrong with that, but in the other files we use "behavior" so better to make it consistent.

> _[Optional] If this a change to a current behaviour, epxlain what's the current behaviour._

Reviewer: ```suggestion
_[Optional] If this is a change to a current behavior, explain what the current behavior is._
```
